src/MenuLinkTreeOverride.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\stanford_profile_helper;
+
+use Drupal\Core\Menu\MenuLinkTreeInterface;
+use Drupal\Core\Menu\MenuTreeParameters;
+
+/**
+ * Service decorator for the menu.link_tree service.
+ */
+class MenuLinkTreeOverride implements MenuLinkTreeInterface {
+
+  /**
+   * Original Menu Tree service.
+   *
+   * @var \Drupal\Core\Menu\MenuLinkTreeInterface
+   */
+  protected $menuTree;
+
+  /**
+   * Menu Tree service override constructor.
+   *
+   * @param \Drupal\Core\Menu\MenuLinkTreeInterface $menu_tree
+   *   Original Menu Tree service.
+   */
+  public function __construct(MenuLinkTreeInterface $menu_tree) {
+    $this->menuTree = $menu_tree;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function build(array $tree) {
+    $build = $this->menuTree->build($tree);
```

Comment:
if you extend the MenuLinkTree object, this line would change to `$build = parent::build($tree);`